File: androidApp/src/main/kotlin/com/anticosmic/android/ui/components/CosmicButton.kt
package com.anticosmic.android.ui.components

import androidx.compose.foundation.layout.PaddingValues
import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.material3.Button
import androidx.compose.material3.ButtonDefaults
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.unit.dp
import com.anticosmic.android.ui.theme.CosmicBlue
import com.anticosmic.android.ui.theme.White

@Composable
fun CosmicButton(
    text: String,
    onClick: () -> Unit,
    modifier: Modifier = Modifier
) {
    Button(
        onClick = onClick,
        modifier = modifier,
        colors = ButtonDefaults.buttonColors(
            containerColor = CosmicBlue,
            contentColor = White
        ),
        shape = RoundedCornerShape(50), // Capsule shape
        contentPadding = PaddingValues(horizontal = 32.dp, vertical = 16.dp)
    ) {
        Text(text = text)
    }
}

File: iosApp/iosApp/Components/CosmicButton.swift
import SwiftUI

struct CosmicButton: View {
    let text: String
    let action: () -> Void
    
    var body: some View {
        Button(action: action) {
            Text(text)
                .font(.headline)
                .foregroundColor(.white)
                .padding(.horizontal, 32)
                .padding(.vertical, 16)
                .background(Color(hex: 0x2A5298)) // CosmicBlue
                .clipShape(Capsule())
        }
    }
}
